cli/commands/login/login_test.go

```diff
@@ -48,7 +49,9 @@ func TestMain(m *testing.M) {
 	var (
 		err error
 	)
-	sock, server, _, err = service.StartDedicatedAuthServer(":0")
+
+	sock, server, _, err = service.StartDedicatedAuthServer(":0", func(*grpc.Server) {}, service_auth.WithApiKeySaveOnCreate(false))
```

Comment:
here, 2nd parameter is filled with empty func. In another case, it is set to nil. As mentioned in `auth.go`, consider to make it a service option